controllers/kokumetricsconfig_controller.go

```diff
@@ -274,15 +274,15 @@ func GetAuthSecret(r *KokuMetricsConfigReconciler, kmCfg *kokumetricscfgv1alpha1
 func checkCycle(logger logr.Logger, cycle int64, lastExecution metav1.Time, action string) bool {
 	log := logger.WithValues("KokuMetricsConfig", "checkCycle")
 	if lastExecution.IsZero() {
-		log.Info(fmt.Sprintf("there have been no prior successful %ss to cloud.redhat.com", action))
+		log.Info(fmt.Sprintf("there have been no prior successful %ss", action))
```

Comment:
instead of removing "cloud.redhat.com" we could replace it with the api_url from the CR

Reply:
Since the packaging task now uses this check cycle separately, the log would have said `there have been no prior successful packagings to cloud.redhat.com` which wouldn't make sense since the packages remain in the cluster. Removed the url so that the log makes sense. If we really need to know where the upload is going, we can view the url in the CR.